src/sentry/static/sentry/app/components/modals/sentryAppPublishRequestModal.tsx

```diff
@@ -41,6 +41,7 @@ export default class SentryAppPublishRequestModal extends React.Component<Props>
 
   form = new PublishRequestFormModel();
 
+  //fake
```

Comment:
What's fake?

Reply:
@markstory Whoops, ignore that. Thought I removed it already...